database/migrations/Guild/2024_06_11_215748_peoples_employment_history.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('peoples_employment_history', function (Blueprint $table) {
+            $table->id();
+            $table->integer('peoples_id')->unsigned();
+            $table->integer('apps_id')->unsigned();
+            $table->string('position');
+            $table->decimal('income', 10, 2)->nullable();
+            $table->date('start_date');
```

Comment:
@FredPeal index start date , end date and status, also index for peoples id and apps_id